roles/ceph-common/handlers/main.yml

```diff
@@ -57,20 +58,23 @@
   when:
     socket.rc == 0 and
     ansible_distribution == 'Ubuntu' and
+    not use_systemd and
     osd_group_name in group_names
 
 - name: restart ceph mdss on ubuntu
   command: initctl restart ceph-mds cluster={{ cluster }} id={{ ansible_hostname }}
   when:
     socket.rc == 0 and
     ansible_distribution == 'Ubuntu' and
+    not use_systemd and
     mds_group_name in group_names
 
 - name: restart ceph mdss
   command: service ceph restart mds
   when:
     socket.rc == 0 and
     ansible_distribution != 'Ubuntu' and
+    not use_systemd and
     mds_group_name in group_names and
     ceph_stable and
     ceph_stable_release in ceph_stable_releases
```

Comment:
When this code was first introduced (https://github.com/ceph/ceph-ansible/commit/e6a97274786ea1b1d51a46a592ca624fcaf625a7), it would have kept this command from being run on infernalis. As it is currently this command would run on infernalis if ``ceph_stable`` was true. I think we do want to use ``is_before_infernalis`` here.

You can see here that infernalis is not in ``ceph_stable_releases`` at that commit:

https://github.com/ceph/ceph-ansible/blob/e6a97274786ea1b1d51a46a592ca624fcaf625a7/roles/ceph-common/defaults/main.yml#L44-L56